Bottom: 1fbaa90ab479a245356f56208024cbb52b9cc8b3
Top:    9d04dc6edb4b665939be85417c15efa80aaa4da6
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-12 11:12:25 -0500

Create certificate directories during image build


---
diff --git a/docker/docker-mediawiki/configure.sh b/docker/docker-mediawiki/configure.sh
index 06ed2991..eba1977c 100755
--- a/docker/docker-mediawiki/configure.sh
+++ b/docker/docker-mediawiki/configure.sh
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -e
 
+# Create certificate directories
+mkdir -p /etc/grid-security/certificates
+mkdir -p /etc/ssl/certs/
+mkdir -p /etc/ssl/private
+
 # comment out apache2 config file lines that reference the environment variables
 sed -i 's|^Mutex file|#Mutex file|' /etc/apache2/apache2.conf
 sed -i 's|^PidFile |#PidFile |' /etc/apache2/apache2.conf
